apps/cms/src/components/ui/file-upload-dropzone.tsx

```
@@ -0,0 +1,342 @@
+            removeFileFromSet(activeIndex);
+            if (value.length - 1 === 0) {
+              setActiveIndex(-1);
+              return;
+            }
+            movePrev();
+          }
+        } else if (e.key === 'Escape') {
+          setActiveIndex(-1);
+        }
+      },
+      // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)